src/config.h.in

```diff
@@ -5,26 +5,16 @@
 extern "C" {
 #endif
 
-#cmakedefine HAVE_STDBOOL_H
-
-#ifdef HAVE_STDBOOL_H
+#if __has_include(<stdbool.h>)
```

Comment:
This is a C99 requirement, however `__has_include` is not a standard macro.  Perhaps conditionalize it on the C standard?

```suggestion
#if __STDC_VERSION__ >= 199901l
```